examples/python/trigger-pagerduty-incident/handler/handler.py

```diff
@@ -63,94 +69,112 @@ def invoke(self,obj):
         Returns:
             FaaSResponse -- status code and message
         """
-        obj['routing_key'] = self.routingkey
-        obj['event_action'] = self.event_action
         try:
             resp = self.session.post(PAGERDUTY_API_PATH,json=obj)
             resp.raise_for_status()
-            print(resp.text)
+            debug(f'{bgc.OKGREEN}Response Body > {bgc.ENDC}{resp.text}')
             resp_body = json.loads(resp.text)
-            return FaaSResponse('200', 'Successfully {0}ed event action with dedup_key: {1}'.format(self.event_action, resp_body['dedup_key']))
+            return FaaSResponse('200', 'Successfully triggered event action with dedup_key: {0}'.format(resp_body['dedup_key']))
         except requests.HTTPError as err:
             return FaaSResponse('500', 'Could not invoke Pagerduty API > HTTPError: {0}'.format(err))
 
 def handle(req):
     # Validate Event input
     try:
         cevent = json.loads(req)
-    except ValueError as err:
-        res = FaaSResponse('400','invalid JSON > ValueError: {0}'.format(err))
+    except json.decoder.JSONDecodeError as err:
+        res = FaaSResponse('400','Invalid JSON > JSONDecodeError: {0}'.format(err))
         print(json.dumps(vars(res)))
+        debug(f'{bgc.OKBLUE}Event > {bgc.ENDC}{req}')
```

Comment:
Same question as above regarding OpenFaaS/Kubernetes interpretation of stderr output. Whatever is printed here, OpenFaaS will only interpret this section `print(json.dumps(vars(res)))` to indicate success/failure? I'd rather inverse the logic, i.e. print debug before the final `print` and `return`.

Reply:
OpenFaaS by default combines the stdout and stderr to the http response. However, there is a env variable `combine_output: false` that prevents the stderr from showing up on http response back to OpenFaaS and is only logged on the console. 

I have it ordered this way to help show the error first and then supporting data below the error for context.

Reply:
I've  reordered and updated the debug messaging now.